test/Table_Tests/src/In_Memory/Table_Running_Spec.enso

```diff
@@ -88,6 +89,218 @@ add_specs suite_builder =
+            result = data.table.running Statistic.Minimum "Ticket Price"
+            expected_column = Column.from_vector "Running Minimum of Ticket Price" [100.5, 100.5, 73.23, 73.23, 73.23]
+            #   | Flight | Passenger | Ticket Price | Running Minimum of Ticket Price
+            #---+--------+-----------+--------------+-------------------------
+            # 0 | BA0123 | A         | 100.5        | 100.5
+            # 1 | BA0123 | B         | 575.99       | 100.5
+            # 2 | SG0456 | A         | 73.23        | 73.23
+            # 3 | BA0123 | C         | 112.34       | 73.23
+            # 4 | SG0456 | E         | 73.77        | 73.23
+            expected_table = data.table.zip expected_column
+            result.should_equal expected_table
+    suite_builder.group "nothing handling" group_builder->
+        #   | Flight | Passenger | Ticket Price
+        #---+--------+-----------+--------------
+        # 0 | BA0123 | A         | 100.5
+        # 1 | BA0123 | B         | 575.99
+        # 2 | SG0456 | A         | nothing
+        # 3 | BA0123 | C         | nothing
+        # 4 | SG0456 | E         | 73.77
+        flight = ["Flight", ["BA0123", "BA0123", "SG0456", "BA0123", "SG0456"]]
+        passenger = ["Passenger", ["A", "B", "A", "C", "E"]]
+        ticket_price = ["Ticket Price", [100.50, 575.99, Nothing, Nothing, 73.77]]
+        table = Table.new [flight, passenger, ticket_price]
+        group_builder.specify "Running count doesn't care about nothing values" <|
+            result = table.running Statistic.Count "Passenger"
+            expected_column = Column.from_vector "Running Count of Passenger" [1, 2, 3, 4, 5]
+            #   | Flight | Passenger | Ticket Price | Running Count of Passenger
+            #---+--------+-----------+--------------+-------------------------
+            # 0 | BA0123 | A         | 100.5        | 1
+            # 1 | BA0123 | B         | 575.99       | 2
+            # 2 | SG0456 | A         | nothing      | 3
+            # 3 | BA0123 | C         | nothing      | 4
+            # 4 | SG0456 | E         | 73.77        | 5
+            expected_table = table.zip expected_column
+            result.should_equal expected_table
+        group_builder.specify "Running sum works with nothing values" <|
+            result = table.running Statistic.Sum "Ticket Price"
+            expected_column = Column.from_vector "Running Sum of Ticket Price" [100.5, 676.49, Nothing, Nothing, Nothing]
+            #   | Flight | Passenger | Ticket Price | Running Sum of Ticket Price
+            #---+--------+-----------+--------------+-------------------------
+            # 0 | BA0123 | A         | 100.5        | 100.5
+            # 1 | BA0123 | B         | 575.99       | 575.99
+            # 2 | SG0456 | A         | Nothing      | Nothing
+            # 3 | BA0123 | C         | Nothing      | Nothing
+            # 4 | SG0456 | E         | 73.77        | Nothing
```

Comment:
I'm wondering if maybe we should allow the user to choose if presence of Nothing makes all further values Nothing OR if such rows are simply ignored but the running statitic continues to be computed.

What do you think?

Reply:
I don't think we should let the user choose as just adds complication. But aggregate sum ignores nothing, so running sum should match I think.

I'm wondering if both should give a warning, but not sure...

Reply:
I think a warning sounds good, although I'm not sure if not too annoying.

Such a warning should be controllable by the `on_problems` setting - if someone wants to error they can override to `Report_Error` and also they can `Ignore` it. So IMHO that could work nicely.

Reply:
Ah yes! I was worried about the annoyingness. But you are right the on_problem fixes this nicely. I will make that change in this MR for running.

Reply:
Done without the warning which I will add in next MR